arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/Resource.kt

```diff
@@ -159,8 +161,8 @@ sealed class Resource<out A> {
   fun <B> zip(other: Resource<B>): Resource<Pair<A, B>> =
     map2(other, ::Pair)
 
-  suspend inline fun <B> foldLeft(inital: B, crossinline f: (B, A) -> B): B =
-    use { f(inital, it) }
+  fun <B> product(other: Resource<B>): Resource<Tuple2<A, B>> =
```

Comment:
This is currently duplicated 🤔 but for `Pair` and `Tuple2`.
`product` vs `zip`..

Reply:
Removed product and used zip.